tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -33,16 +33,23 @@
 
 Define {
 	MCLK 				24576000
-	NUM_DMICS			0
```

Comment:
I wonder if leaving this OK. Does the NUM_DMICS from CMakeLists.txt override this?

Reply:
kept this